client/src/index.jsx

```diff
@@ -55,24 +68,28 @@ class App extends React.Component {
     return (
       Object.keys(this.state.data[0]).length ?
         <MainContainer>
-          <Title>CONTENT FOR THIS GAME
-            <Browse className='note'>
-              <StyledA href='https://google.com'>
-                {`Browse all `}
-                <em style={{ fontStyle: 'normal' }}>
-                  ({this.state.data.length})
-                </em>
-              </StyledA>
-            </Browse>
-          </Title>
+          <div>
+            <Title>CONTENT FOR THIS GAME
+              <Browse className='note'>
+                <StyledA href='https://google.com'>
+                  {`Browse all `}
+                  <em style={{ fontStyle: 'normal' }}>
```

Comment:
The `<em>` tag makes content italic, but `fontStyle: 'normal'` makes it non-italic. Purpose of this line?